### ISSUE TITLE
Language Files and Assets

### ISSUE THREAD

**Issue by column111** [victorjonsson/jQuery-Form-Validator#645]
Hi, for performance issues I want to use assets, but the problem will be the dynamic loading of the languages. Is there any possibility to solve this or to load all the desired language files at the beginning?

**Comment by victorjonsson** [comment]
You have to explain to me what you mean by "assets". You can use `require` to make include the language files in your bundle. If that's what you're thinking on?

**Comment by column111** [comment]
Yes, I have meant the rails assets pipeline and I have already tried to include the language file by require. That doesn't work but I will try it once again.

**Comment by victorjonsson** [comment]
Well, thats something completely different compared to what I was referring to. 

If I have understood it correctly it's only a matter of where you put the js files in the file system.
